source/hardware/controllers/akai_mpk_mini_2.rst

```diff
@@ -0,0 +1,23 @@
+AKAI MPK Mini mkII
+===================
+
+The *AKAI MPK Mini mkII* is a funny keyboard controller for musicians.
+This device is not really conceived for DJing but this adaptation can be funny.
+
+-  `Manufacturer’s product
+   page <https://www.akaipro.com/mpk-mini-mkii>`__
+
+Mapping description
+-------------------
+
+-  The main keyboard is mapped to the 64 Samplers slots for Skins that have them, since it has 25 keys, upper Samplers can be accessed by pressing the Octave Up/Down buttons.
+-  The 8 pressure sentitive drum pads are attached to 6 Effects, 3 for FX1 and 3 for FX2, the last two pads are mapped for a fourth effect for when it will be available.
```

Comment:
```suggestion
-  The 8 pressure sensitive drum pads are attached to 6 Effects, 3 for FX1 and 3 for FX2, the last two pads are mapped for a fourth effect for when it will be available.
```